modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -4,6 +4,7 @@
 require_relative '../../../models/mobile/v0/adapters/claims_overview'
 require_relative '../../../models/mobile/v0/adapters/claims_overview_errors'
 require_relative '../../../models/mobile/v0/claim_overview'
+require 'parallel'
```

Comment:
parallel is in the gemfile for the test and development groups online.  To make it available here you should move it in the gemfile out of that group.

Reply:
I see that there is a gem called "parallel_tests" would that be the appropriate gem to move?

Reply:
You're right!  I overlooked it.  rubocop and parallel_tests both depend on parallel.  But both of those gems are in the test/development groups.  I'd just add `parallel` to the gemfile outside those groups.  So it'd be a new entry and not a "move".  I think the gems are generally in alphabetical order, so probably on line 91

Reply:
I went ahead and pushed up what I was talking about

Reply:
Awesome thanks